openpype/hosts/standalonepublisher/plugins/publish/validate_shot_duplicates.py

```diff
@@ -17,6 +17,9 @@ def process(self, context):
         shot_names = []
         duplicate_names = []
         for instance in context:
+            if "csv" in instance.data["families"]:
+                continue
+
```

Comment:
Is this also the 'patch' for multiple publishes going into same subset but different version? If so, can we also clean this up.

Either way, if it is or it isn't, can we at least get a comment here as to **WHY** want need to ignore csv family here?

Reply:
I believe since it is standalone publisher this can be ignored - this might not be merged at all.